mozregression/telemetry.py

```diff
@@ -6,41 +7,56 @@
 from pkg_resources import resource_filename
 
 from mozregression import __version__
+from mozregression.dates import is_date_or_datetime, to_datetime
 
 LOG = get_proxy_logger("telemetry")
 PINGS = load_pings(resource_filename(__name__, "pings.yaml"))
 METRICS = load_metrics(resource_filename(__name__, "metrics.yaml"))
 
+UsageMetrics = namedtuple(
+    "UsageMetrics", ["variant", "appname", "build_type", "good", "bad", "launch"]
+)
+
 
 def initialize_telemetry(upload_enabled, allow_multiprocessing=False):
     mozregression_path = Path.home() / ".mozilla" / "mozregression"
     Glean.initialize(
         application_id="org.mozilla.mozregression",
         application_version=__version__,
         upload_enabled=upload_enabled,
-        configuration=Configuration(allow_multiprocessing=allow_multiprocessing),
+        configuration=Configuration(
+            allow_multiprocessing=allow_multiprocessing, ping_tag="mozregression-test"
```

Comment:
Nit: don't forget to remove `ping_tag` before you ship. We're working to make this less weird by allowing configuring debug options using environment vars, see [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1646706).